src/main/java/com/googleinterns/zoomtube/data/Line.java

```diff
@@ -0,0 +1,66 @@
+// Copyright 2020 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package com.googleinterns.zoomtube.data;
+
+import com.google.appengine.api.datastore.Entity;
+import com.google.appengine.api.datastore.Key;
+import com.google.appengine.repackaged.com.google.common.base.Optional;
+import com.google.auto.value.AutoValue;
+import java.util.Date;
+import javax.annotation.Nullable;
+
+/** Contains data pertaining a single line of transcript. */
+@AutoValue
+public abstract class Line {
```

Comment:
I personally think `Line` is a little ambiguous, thoughts on changing name of class to something like `TranscriptLine`?